uefi/flash.go

```diff
@@ -57,7 +64,8 @@ type FlashImage struct {
 	PD   *PDRegion
 
 	// Metadata for extraction and recovery
-	ExtractPath string
+	ExtractPath    string
+	extractRegions []Firmware
```

Comment:
could this just be `regions` or `regionList` ?

Reply:
done!